tests/framework/test_event.py

```diff
@@ -4,72 +4,54 @@
 
 from vivarium.framework.event import Event, EventManager
 
-
-def test_proper_access():
-    ''' Event attributes are meant to be read-only
-    '''
-    idx = pd.Index(range(10))
-    user_data = {'specific_event_data': 'the_data'}
-    e1 = Event(idx, user_data)
-
-    assert (idx == e1.index).all()
-    assert user_data == e1.user_data
-    assert e1.time == None
-    assert e1.step_size == None
-
-    with pytest.raises(AttributeError) as _:
-        e1.index = pd.Index(range(3))
-
-    with pytest.raises(AttributeError) as _:
-        e1.user_data = {'key': 'value'}
-
-    with pytest.raises(AttributeError) as _:
-        e1.time = pd.Timestamp('1/1/2000')
-
-    with pytest.raises(AttributeError) as _:
-        e1.step_size = 43
-
-
 @pytest.fixture
 def event_init():
-    return (
-        (pd.Index(range(10)), {'specific_event_data': 'the_data'}),
-        {
-            'index': pd.Index(range(3)),
+    return {
+        'orig': {
+            'index': pd.Index(range(10)),
             'user_data': {'key': 'value'},
             'time': pd.Timestamp('1/1/2000'),
-            'step_size': 43,
+            'step_size': 12,
+        },
+        'new_val': {
+            'index': pd.Index(range(3)),
+            'user_data': {'new_key': 'new_value'},
+            'time': pd.Timestamp.now(),
+            'step_size': 30,
         }
-    )
+    }
 def test_proper_access2(event_init):
```

Comment:
probably just rename this to ``test_proper_access`` now that you've removed the other test